data/modules/general/threat_modeling.yml

```
@@ -0,0 +1,94 @@
+category: General
+description: Have you performed a theat assessment on your application (aka a Threat Model)?
+minimumRisk: Low Risk
+checkLists:
+  Diagram:
+      - question: We have created a diagram of the software, can tell a story about the system and consider what could go wrong without using words such as "sometimes" or "also".
```

```suggestion
      - question: We have created an architecture or dataflow diagram of the software, can tell a story about the system and consider what could go wrong without using words such as "sometimes" or "also".
```

For the threat modelling we've done, we normally start with an arch diagram and derive a DFD ourselves. Or start with a DFD. Normally we'd expect the teams to be able to provide an arch diagram at the very least